### PR TITLE
ReadTheDocs Update

### PR DESCRIPTION
**Pull request by Seneth**
Reformatted and removed unneeded commands.rst page.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,30 +1,365 @@
...
-, where you can provide your input or find out more ways to help!
+*OMNIBOT is currently under development if you are interested in assisting with the project, hop on the* `OMNIBOT Server <https://discord.gg/38GdPqK>`_  , *where you can provide your input or find out more ways to help!*
 
 OMNIBOT can be found on `GitHub <https://github.com/Yandawl/OMNIBOT>`_
 
-CONTENTS
+Invite Link
+------------------
+
+*Currently the bot is still in development, but keep an eye out for an invite link soon!* **[the following image is a placeholder]**
```
Comment (Yandawl) [inline docs/index.rst]: If you do *invite in chat you'll get an oauth invite URL for the bot

File: docs/index.rst [hunk 2]
```diff
@@ -1,30 +1,365 @@
...
+----------------------
+
+*We do projects like this because we love to build things that people will love and find useful, but if you would like to support us, a cup of coffee is always nice!*
+
+.. image:: https://az743702.vo.msecnd.net/cdn/kofi4.png?v=f
+   :target: https://ko-fi.com/A752JVZ
+
+
+============
+Commands
```
Comment (Yandawl) [inline docs/index.rst]: Commands  should be contained in a separate file

File: docs/index.rst [hunk 3]
```diff
@@ -1,30 +1,365 @@
...
+-----
+**Everyone**
+
+- **\*char [world] [forename] [surname]** *saves your FFXIV character*
+- **\*removechar** *removes your FFXIV character*
+- **\*whochar [@User]** *gets information about a user's FFXIV character*
+- **\*whologs [@User]** *gets a user's FFLogs data*
+
+============
+F.A.Q.
```
Comment (Yandawl) [inline docs/index.rst]: FAQ should be contained in a separate file

**Comment by Seneth** [comment]
Split the files as requested.

**Pull request merged**